tiles/src/main/java/com/protomaps/basemap/layers/Places.java

```diff
@@ -66,10 +86,13 @@ public void processNe(SourceFeature sf, FeatureCollector features) {
           theme_max_zoom)
         .setAttr("pmap:kind", kind)
         .setAttr("pmap:kind_detail", kind_detail)
-        .setAttr("population", sf.getString("pop_max"))
-        .setAttr("population_rank", sf.getString("rank_max"))
+        .setAttr("population", parseIntOrNull(sf.getString("pop_max")))
+        .setAttr("pmap:population_rank", parseIntOrNull(sf.getString("rank_max")))
         .setAttr("wikidata_id", sf.getString("wikidata"))
-        .setBufferPixels(128);
+        .setBufferPixels(128)
+        // we set the sort keys so the label grid can be sorted predictably (bonus: tile features also sorted)
+        //.setSortKey(getSortKey("pmap:min_zoom",  "population_rank", "population", "name"))
```

Comment:
@bdon once the sort key function and Java type conversions are fixed, this can be re-enabled.